libr/core/cmd_anal.c

```diff
@@ -3089,11 +3089,23 @@ static ut64 initializeEsil(RCore *core) {
 	return addr;
 }
 
+static const ut8 *resize_read_buf(RAnal *anal, int new_len) {
+	RCore *core = anal->user;
+	ut8 *buf = malloc (new_len);
+	if (!buf) {
+		return NULL;
+	}
+	const char *name = r_reg_get_name (anal->reg, R_REG_NAME_PC);
+	ut64 addr = r_reg_getv (anal->reg, name);
+	(void)r_io_read_at (core->io, addr, buf, new_len);
```

Comment:
beter use calloc for buf if you dont check return value of read

Reply:
done